src/domain/entities/EditorTool.ts

```diff
@@ -38,3 +38,33 @@ export default interface EditorTool {
     cdn?: string;
   };
 }
+
+/**
+ * Editor tool with user binding
+ */
+export interface EditorToolWithUserBinding extends EditorTool {
+  /**
+   * Is tool included in user's editor settings
+   */
+  isInstalled: boolean;
+}
+
+/**
+ * Data which is returned when tool is removed
+ */
+export interface RemovedToolData {
+  /**
+   * Removed tool identifier
+   */
+  removedId: EditorTool['id'];
+}
+
+/**
+ * Data which is returned when tool is added
+ */
+export interface AddedToolData {
```

Comment:
do we really need to create separate types for it?

Reply:
removed